src/components/MessageHTMLBody.vue

```diff
@@ -67,7 +67,7 @@ export default {
 				// send css vars to client page
 				this.$refs.iframe.iFrameResizer.sendMessage({
 					cssVars: {
-						...getCssVar('--color-main-text'),
+						...getCssVar('--color-main-background'),
```

Comment:
Just sending another css variable to the client page won't change anything on its own. If we scrap the injection of the text color and just force a white background, this whole block can be removed (including the `getCssVar()` method).

See the injected css for reference (`css/html-response.css`):
```css
* {
	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Cantarell, Ubuntu, 'Helvetica Neue', Arial, 'Noto Color Emoji', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
}

body {
	color: var(--color-main-text);
}
```

The whole body block can be removed aswell from the css file.